test/fixtures/testpkgstophook/plan.ps1

```diff
@@ -0,0 +1,11 @@
+$pkg_name="testpkgstophook"
+$pkg_origin="habitat-testing"
+$pkg_version="0.1.0"
+$pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+$pkg_license=@("Apache-2.0")
+
+function Invoke-Install {
+    mkdir "$pkg_prefix\hooks"
+    Set-Content -Path "$pkg_prefix\hooks\run" -Value "try { while (`$true) { Start-Sleep 1 }} finally { 'run hook is terminating' | Out-File -path `$env:temp/testpkgstophook.out}"
+    Set-Content -Path "$pkg_prefix\hooks\post-stop" -Value "'post-stop hook has fired' | Out-File -path `$env:temp/testpkgstophook.out -Append"
```

Comment:
I have always assumed you could only write powershell hooks on windows but I *think* if you include powershell as a runtime dep and use a pwsh shebang. This might just work. will try it.

Reply:
I did that for the [`simple-hooks`](https://github.com/habitat-sh/habitat/blob/master/test/fixtures/plans/simple-hooks/hooks/run) plan. Although looking at it now I realize I forgot to include powershell as a runtime dep, but it works since we only ever run the package as an e2e tests which always sets up powershell.

Reply:
ugh. sadly powershell does not gracefully handle SIGTERM so the hook behaves differently, Looks like there is an [issue for this](https://github.com/PowerShell/PowerShell/issues/1040).

Reply:
Bummer. Thanks for looking into it.